src/signals/incident-management/containers/Dashboard/styled.ts

```diff
@@ -1,10 +1,24 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { themeColor } from '@amsterdam/asc-ui'
+import { breakpoint, themeColor, themeSpacing } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 
 export const StyledRow = styled.div`
   width: 100%;
   background-color: ${themeColor('tint', 'level2')};
   margin: 0;
 `
+
+export const Wrapper = styled.div`
```

Comment:
Wrapper is not used anymore. Replace the Row component.

Reply:
Did it.